src/workspaces/graphql/queries.graphql

```diff
@@ -0,0 +1,44 @@
+query WorkspacesPage($page: Int, $perPage: Int) {
+  workspaces(page: $page, perPage: $perPage) {
+    totalItems
+    items {
+      id
+      name
+      countries {
+        code
+        flag
+      }
+    }
+  }
+}
+
+query WorkspacePage($id: String!, $page: Int, $perPage: Int) {
```

Comment:
You have to add the fragments for the dialogs and to remove all fields non-used on the page directly.